appveyor.yml

```diff
@@ -0,0 +1,50 @@
+build: false
+
+environment:
+  matrix:
+    - PYTHON_VERSION: 2.7
+      PYTHON_SVER: 27
+      MINICONDA: C:\Miniconda
+    - PYTHON_VERSION: 3.4
```

Comment:
Why not test on 3.5 or 3.6 (or both)? Appveyor does offer both:
https://www.appveyor.com/docs/build-environment/#miniconda
Then we could have numpy 1.12 and avoid dropping those doctests.